### PR TITLE
PKS Getting Started

### PR DESCRIPTION
**Pull request by bmansheim**
Created draft PKS getting started an article for review

### CODE REVIEW

File: content/platforms/pks/_index.md [hunk 1]
```diff
@@ -0,0 +1,598 @@
...
+        name: standard
+    mountOptions:
+        - debug
+    parameters:
+        type: pd-standard
+    provisioner: kubernetes.io/gce-pd
+    reclaimPolicy: Retain
+    ```
+
+1. Create the appropriate yaml file to you IaS and apply it:
```
Comment (bmansheim) [inline content/platforms/pks/_index.md]: This section needs some formatting improvement. I don't understand what is supposed to be commented.
You can see how it looks at: https://docs.redislabs.com/staging/pks-getting-started/platforms/pks/#step-2-get-and-prepare-deployment-files

Comment (bmansheim) [reply content/platforms/pks/_index.md]: @amiramm I updated the indentation and it looks to me now to be the same as it was when you committed. Let me know if that's not correct.

Do you want uiServiceType and persistentSpec to be commented out?

File: content/platforms/pks/_index.md [hunk 2]
```diff
@@ -294,7 +294,7 @@ For production environments, make sure that Persistent Volume Claims (PVCs) are
     <!-- - persistentSpec: enabled: \<false/true\>
     Check your Redis Software nodes’ enabled/disabled flag for [persistency](https://redislabs.com/redis-features/persistence). The default is “false.”
     we now default to using persistence-->
-    - `storageClassName:` “<span style="color: #ff0000;">your storage class name from the previous step</span>“
+    - `storageClassName:` Your storage class name from the previous step
```
Comment (bmansheim) [inline content/platforms/pks/_index.md]: @amiramm Is this correct?

Comment (amiramm) [reply content/platforms/pks/_index.md]: yes

File: content/platforms/pks/_index.md (hunk 1 above)
Comment (amiramm) [reply content/platforms/pks/_index.md]: @bmansheim 
These are meant to be code blocks, contained within ```. They appear as plain text for some reason.
Screen Shot 2019-07-30 at 9 44 18 PM

In this case, indentation is off.
Screen Shot 2019-07-30 at 9 44 48 PM

Can you help with these, please?

Comment (amiramm) [reply content/platforms/pks/_index.md]: >Do you want uiServiceType and persistentSpec to be commented out?
no, those are fine now.

**Comment by bmansheim** [comment]
Hi Amiram. Do you want to give a final OK on this before I publish it?

**Pull request merged**